src/vsphere_cpi/spec/unit/cloud/vsphere/vm_config_spec.rb

```diff
@@ -266,7 +266,8 @@ def fake_datastore(name, free_space, mob = nil)
             accessible_datastores: {
               'smaller-ds'=> smaller_ds,
             },
-            mob: cluster1_mob
+            mob: cluster1_mob,
+            host: [host_system],
```

Comment:
then host can be removed from cluster1_mob and cluster2_mob definition as it is not being used

Reply:
It is being used. Caused error when i removed.